src/test/unit_tests/test_bucketspace_fs.js

```diff
@@ -421,7 +425,16 @@ mocha.describe('bucketspace_fs', function() {
 async function create_bucket(bucket_name) {
     const param = { name: bucket_name};
     await bucketspace_fs.create_bucket(param, dummy_object_sdk);
-    const bucket_config_path = path.join(config_root, buckets, param.name + '.json');
+    const bucket_config_path = get_config_file_path(buckets, param.name);
     const stat1 = await fs.promises.stat(bucket_config_path);
     assert.equal(stat1.nlink, 1);
 }
+
+
+function get_config_file_path(config_type_path, file_name) {
+    return path.join(config_root, config_type_path, file_name + '.json');
+}
+
+function get_access_key_config_file_path(config_type_path, file_name) {
```

Comment:
Can you write a comment?
and also maybe rename the function?
I find the name confusing when the function reurn `.symlink`

Reply:
rename to what? not sure what's confusing

Reply:
By the code, I will assume we are getting the symlink path and not the actual file path.
This is the reason I think a comment here is needed, and if it is the case indeed the `get_access_key_config_file_path` is not accurate (but something that involves symlink maybe will do it)